itests/org.openhab.binding.mqtt.homie.tests/src/main/java/org/openhab/binding/mqtt/HomieImplementationTest.java

```diff
@@ -156,7 +156,7 @@ public void setUp() throws InterruptedException, ExecutionException, TimeoutExce
     public void tearDown() throws InterruptedException, ExecutionException, TimeoutException {
         if (connection != null) {
             connection.removeConnectionObserver(failIfChange);
-            connection.stop().get(500, TimeUnit.MILLISECONDS);
+            connection.stop().get(1000, TimeUnit.MILLISECONDS);
```

Comment:
This change does not seem to be related to this PR - if you want to change this code, please do it in a separate PR for better traceability.